Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -1,37 +1,472 @@
-﻿namespace BootCamp.Chapter
+﻿using System.Globalization;
+using System.Text;
+using System;
+using System.Diagnostics;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4971928)